src/error.ts

```diff
@@ -0,0 +1,34 @@
+import { defaultTo } from "lodash";
+
+interface FirebaseErrorOptions {
```

Comment:
This should probably be exported.

Reply:
What’s the reasoning behind this? This error is used across the entire project and I didn’t see a use case where it would be needed. I find it a good practice to keep the API minimal.

Reply:
Ah - nevermind. I think my thinking was that it had to be exported so that errors would show up correctly, but I think it works fine.